README.md

```diff
@@ -15,7 +15,12 @@ This library is the canonical location of the Open Cluster Management API defini
 
 ## Consumers
 
-Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. For example, see [registration-operator](https://github.com/open-cluster-management-io/registration-operator) for how the `api` is being consumed.
+Various projects under [Open Cluster Management](https://github.com/open-cluster-management-io) leverage this `api` library. The following are relationships between api and consumers.
+*  `ManagedCluster`, `ManagedClusterSet`,`ClusterClaim`: implemented in [registration](https://github.com/open-cluster-management-io/registration).
```

Comment:
for cluster registration and lifecycling.

Reply:
Done.